### PR TITLE
Add support for JSON dashboards

### PR DESCRIPTION
**Pull request by thouveng**

### CODE REVIEW

File: _states/grafana3_dashboard.py [hunk 1]
```diff
@@ -88,15 +89,29 @@ def present(name,
...
-    base_dashboards_from_pillar = base_dashboards_from_pillar or []
-    base_panels_from_pillar = base_panels_from_pillar or []
-    base_rows_from_pillar = base_rows_from_pillar or []
     dashboard = dashboard or {}
 
     if isinstance(profile, six.string_types):
         profile = __salt__['config.option'](profile)
 
+    if dashboard_format == 'json':
+        # The dashboard is a JSON
```
Comment (simonpasquier) [inline _states/grafana3_dashboard.py]: (nit) This is a raw JSON dashboard

File: _states/grafana3_dashboard.py [hunk 2]
```diff
@@ -65,6 +65,7 @@ def present(name,
             base_panels_from_pillar=None,
             base_rows_from_pillar=None,
             dashboard=None,
+            dashboard_format='yaml',
```
Comment (simonpasquier) [inline _states/grafana3_dashboard.py]: please update the comments below to include the new parameter

File: metadata/service/collector.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+applications:
+- grafana
+classes:
+- service.grafana.support
```
Comment (simonpasquier) [inline metadata/service/collector.yml]: not needed

File: _states/grafana3_dashboard.py (hunk 2 above)
Comment (thouveng) [inline _states/grafana3_dashboard.py]: done

Comment (thouveng) [inline _states/grafana3_dashboard.py]: done

**Pull request merged**